public class FunctionUse4408 {

    public void funcUse() {
		org.apache.hadoop.fs.PathIOException pathioexception = new org.apache.hadoop.fs.PathIOException();
		pathioexception.getPath(null);
    }
}